atomix/cluster/src/test/java/io/atomix/raft/RaftFailOverTest.java

```diff
@@ -173,8 +173,8 @@ public void shouldReplicateSnapshotOnJoin() throws Exception {
   public void shouldReplicateSnapshotWithManyFilesOnJoin() throws Exception {
     // given
     final var follower = raftRule.shutdownFollower();
-    raftRule.appendEntries(20);
-    final long snapshotIndex = 10L;
+    raftRule.appendEntries(30);
```

Comment:
:thought_balloon: Might make sense to squash this with the previous commit, since they both highly dependent on each other. But just a thought.

Reply:
Yeah, makes sense :+1: